test/smoke-test.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+set -xe
+
+# Wait for services to come up
+docker run --net host -e TARGETS=127.0.0.1:3306,127.0.0.1:6385,127.0.0.1:80 -e TIMEOUT=60 waisbrot/wait
+
+# Check containers are running
+for CONTAINER in mariadb ironic httpd dnsmasq
+do
+  RESULT=$(docker inspect -f "{{.State.Running}}" $CONTAINER)
+  if [ "$RESULT" != "true" ]
+  then
+    docker logs $CONTAINER
+    echo "$CONTAINER is not running."
+    exit 1
+  fi
+done
+
+# Ensure Ironic Is Responding
+curl -H "X-OpenStack-Ironic-API-Version: 1.52" 'http://localhost:6385/v1/nodes'
+
```

Comment:
bonus point for reading `http://localhost:6385/v1/drivers` and making sure it's non-empty. this way we'll verify both API and conductor (your approach only verifies API)

Reply:
This is redundant now that you're checking `/drivers`